packages/auth/src/session.ts

```diff
@@ -257,48 +261,22 @@ export class Session {
     const proof = new Proof({
       address: this.account.address
     })
+
     proof.claims.app = this.name
     proof.setExpiryIn(this.expiration)
 
     const ethAuth = new ETHAuth()
-    const configFinder = new SequenceUtilsFinder(this.authProvider)
-    const authWallet = this.account.authWallet()
     const expiration = this.now() + this.expiration - EXPIRATION_JWT_MARGIN
 
+    // Sign proof message using account
     const proofString = {
-      // Fetch latest config
-      // TODO: Should only search for latest config if necessary to be more efficient.
-      //       Perhaps compare local config hash with on-chain hash before doing
-      //       the search through the logs. Should do this accross sequence.js
-      proofString: configFinder
-        .findCurrentConfig({
-          address: authWallet.wallet.address,
-          provider: this.authProvider,
-          context: authWallet.wallet.context,
-          knownConfigs: [authWallet.wallet.config]
-        })
-        .then(val => {
-          if (!val.config) throw Error("Can't find latest config")
-          return authWallet.wallet
-            .useConfig(val.config!)
-            .sign(proof.messageDigest())
-            .then(signature => {
-              const decodedSignature = decodeSignature(signature)
-              const totalWeight = decodedSignature.signers.filter(isDecodedSigner).reduce((totalWeight, signer) => totalWeight + signer.weight, 0)
-              if (totalWeight < decodedSignature.threshold) {
-                throw Error(`insufficient signing power, need ${decodedSignature.threshold}, have ${totalWeight}`)
-              }
```

Comment:
Is the signature guaranteed to meet threshold? This was a workaround for https://github.com/0xsequence/wallet-webapp/pull/328 IIRC, so it's nice if it could be removed, but just want to confirm that lazy octopus fixes this.